docs/DeveloperGuide.md

```diff
@@ -43,41 +43,47 @@ import sys
 
 from jsonargparse.typing import PositiveInt
 
+from data_juicer.utils.constant import Fields, StatsKeys
+
```

Comment:
Maybe a new step before step 1 should be added to teach users to add a new StatsKeys in `utils.constant.py`